package.json

```diff
@@ -42,6 +45,7 @@
     "@types/papaparse": "^5.3.7",
     "@types/react": "^18.1.0",
     "@types/react-dom": "^18.2.6",
+    "@types/styled-components": "^5.1.26",
```

Comment:
There is no need to use styled-components, could use import { styled } from '@mui/material';